Fix missing placeholder images for news articles

KludgeKML opened the pull request:
## What

Reinstate placeholder images for news articles using the NewsImage concern.

Audit Trail:
https://github.com/alphagov/government-frontend/blob/168198215859d8f9813dee2fd80206daac7af725/app/presenters/content_item/news_image.rb#L19

https://trello.com/c/oCO6I8lP/355-fix-missing-placeholder-new-image, [Jira issue PNP-6350](https://gov-uk.atlassian.net/browse/PNP-6350)

sairamya93 commented:
Can we include the `news_image` concern in the `NewsArticle` model and have the `news_article` tests?
https://github.com/alphagov/frontend/blob/main/app/models/concerns/news_image.rb

deborahchua reviewed (approved):
LGTM! I've left one comment on updating the commit description.

deborahchua commented:
> Can we include the `news_image` concern in the `NewsArticle` model and have the `news_article` tests? https://github.com/alphagov/frontend/blob/main/app/models/concerns/news_image.rb

I don't think we need to include it as a concern as it's only used in news articles, so the code isn't shared with any other document types.

KludgeKML commented:
> > Can we include the `news_image` concern in the `NewsArticle` model and have the `news_article` tests? https://github.com/alphagov/frontend/blob/main/app/models/concerns/news_image.rb
> 
> I don't think we need to include it as a concern as it's only used in news articles, so the code isn't shared with any other document types.

I think what @sairamya93 is saying is that it's already a concern (which the Speech model uses)

KludgeKML committed:
fix: reinstate placeholder/default images in news_articles

- These appear to have been missed in the news_article move. Fortunately the
  speech documents have the pre-existing concern and shared tests for NewsImages.

KludgeKML commented on spec/models/news_article_spec.rb:
Since we're using the existing NewsImage concern brought in with Speech, I've simplified this PR a lot and the code that brings in the concern and shared examples has the audit trail.

sairamya93 commented:
Yes we do have `news_image` for Speech and we have already removed the references in government-frontend for that. Think `news articles` and `Speech` only use that concern

deborahchua commented:
> Yes we do have `news_image` for Speech and we have already removed the references in government-frontend for that. Think `news articles` and `Speech` only use that concern

Ah sorry! 👍

deborahchua reviewed (approved):
🎉
